src/main/java/com/amazon/opendistroforelasticsearch/ad/rest/AbstractSearchAction.java

```diff
@@ -68,15 +69,23 @@
 
     private final String index;
     private final Class<T> clazz;
-    private final String urlPath;
+    private final List<String> urlPaths;
+    private final List<Pair<String, String>> replacedPaths;
```

Comment:
These should be called `deprecatedPaths` I think.

Reply:
And these are also `urlPaths`, right? So maybe `deprecatedUrlPaths`?

Reply:
Yeah definitely, nice catch :)